.gitlab-ci.yml

```diff
@@ -0,0 +1,16 @@
+#variables:
```

Comment:
Couple nits...

1. Let's remove commented out things we aren't using...
2. Typical indent for yaml is 2 spaces, not 4. We can add yamllint to validate for that in a separate PR.